pnet_sys/src/unix.rs

```diff
@@ -28,7 +28,46 @@ pub mod public {
     pub const IP_HDRINCL: libc::c_int = libc::IP_HDRINCL;
 
     pub const IFF_LOOPBACK: libc::c_int = libc::IFF_LOOPBACK;
+    
+    #[cfg(target_os = "macos")]
+    mod device_flags {
```

Comment:
Do these constants definitely not exist in the standard library?

Reply:
Would be awesome if we can find that. Not optimal to have to maintain them ourselves.

Reply:
Now @LuoZijun got them merged in libc! Which is great! Only problem is that a version with them in has not been released to crates.io yet. Since libc do fairly frequent releases. Judging from the recent history it's more often than every month. I would suggest we maybe wait with merging this PR until we can use those constants in libc?

Until then there is still Windows support to implement to get rid of the conditional compilation for example.